commerce_coordinator/apps/titan/pipeline.py

```diff
@@ -218,3 +218,30 @@ def run_filter(self, order_uuid, **kwargs):  # pylint: disable=arguments-differ
         update_billing_address_output = BillingAddressSerializer(data=response)
         update_billing_address_output.is_valid(raise_exception=True)
         return {'billing_address_data': update_billing_address_output.data}
+
+
+class UpdateTitanPayment(PipelineStep):
+    """
+    Updates a payment in Titan
+    """
+
+    def run_filter(
+            self,
+            payment_number,
+            payment_state,
+            response_code
+    ):  # pylint: disable=arguments-differ
+
+        api_client = TitanAPIClient()
+        try:
+            response = api_client.update_payment(
+                payment_number=payment_number,
+                payment_state=payment_state,
+                response_code=response_code)
+
+        except HTTPError as exc:
+            raise APIException("Error updating the payment details in titan") from exc
```

Comment:
Is there enough information in the exception? Will there a way to connect the exception message with the parameters used to run the PipelineStep? I noticed many other PipelineSteps in the code have a `logger.exception` with the parameters of PipelineStep. Does it make sense to reproduce here?

Reply:
added